ipywidgets/interact.py

```diff
@@ -63,13 +90,13 @@ class StaticInteract(object):
          }}
       }}
     </script>
-    
-    <div>
+
+    <div id="static_widget">
+      {widgets} <!--Same order as ipywidgets-->
```

Comment:
I changed the order to make it look more like ipywidgets: First the widgets then the plot.